.github/workflows/main.yml

```diff
@@ -0,0 +1,405 @@
+name: Build & Test
+
+on:
+  # Trigger on push to branches `master` and `3.5`.
+  push:
+    branches: [ master, 3.5 ]
+  # Trigger on pull request events for PRs that have `master` as their target branch
+  pull_request:
+    branches: [ master ]
+
+  # Allow running the workflow manually
+  workflow_dispatch:
+
+defaults:
+  run:
+    # GitHub Actions adds `errexit` and `pipefail` by default, but we add `xtrace`
+    # to improve debugging some of the longer scripts.
+    shell: /bin/bash -o errexit -o pipefail -o xtrace {0}
+
+jobs:
+  main:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-20.04
+    name: ${{ matrix.test_name }}
+
+    strategy:
+      # Let other jobs continue even if one of them fails
+      fail-fast: false
+      matrix:
+        test_name:
+          - "codecov-lua5.3"
+          - "coveralls-lua5.2"
+          - "lua5.1"
+          - "luajit"
+          - "fixed-lgi-lua5.2"
+        include:
+          - test_name: "codecov-lua5.3"
+            lua_version: "5.3"
+            lua_name: "lua5.3"
+            coverage: "codecov"
+            manual_screens: true
+
+          - test_name: "coveralls-lua5.2"
+            lua_version: "5.2"
+            lua_name: "lua5.2"
+            coverage: "coveralls"
+            manual_screens: true
+
+          - test_name: "lua5.1"
+            lua_version: "5.1"
+            lua_name: "lua5.1"
+
+          - test_name: "luajit"
+            lua_version: "5.1"
+            lua_name: "luajit"
+            lua_library: "/usr/lib/x86_64-linux-gnu/libluajit-5.1.so"
+            lua_include: "/usr/include/luajit-2.1"
+            luarocks_args: "--lua-suffix=jit-2.1.0-beta3"
```

Comment:
Why is there differences in the used versions? I never used luajit before, so there is maybe something I'm missing...

Reply:
You mean the difference compared to Travis?

I figured that if we update the OS by two versions, the package is likely to update as well. `2.1.0-beta3` is the version reported by `apt-cache`.

Reply:
Nahh, I was just wondering myself why we use lua_library jit-5.1 with lua_include jit-2.1 and a lua-suffix jit-2.1.0-beta3. Why aren't these numbers more homogeneous?

Please dismiss this comment, tho. I'm not familiar with luajit and looking at the AUR PKGBUILD for [awesome-luajit-git](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=awesome-luajit-git), I guess it's the normal behavior. :shrug: